pom.xml

```diff
@@ -87,7 +87,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>16.0.1</version>
+            <version>18.0</version>
```

Comment:
like above, this is managed (though this time correctly) in open-source-parent; you can just bump that to 1.1.8 and remove this version declaration

Reply:
👍